internal/dispatch/caching/caching.go

```diff
@@ -61,6 +62,8 @@ func NewCachingDispatcher(
 			BufferItems: 64,      // number of keys per Get buffer.
 			Metrics:     true,    // collect metrics.
 		}
+	} else {
```

Comment:
should we just always log this even if it's default?

Reply:
It was making the testing output longer